apps/demo-game/src/pages/play/cockpit.tsx

```diff
@@ -1,11 +1,15 @@
 import { useMutation, useQuery } from '@apollo/client'
 import { Layout } from '@gbl-uzh/ui'
-import { Switch, Table } from '@uzh-bf/design-system'
+import { CycleCountdown, Switch, Table } from '@uzh-bf/design-system'
 import {
   ChartContainer,
   ChartTooltip,
   ChartTooltipContent,
 } from '@uzh-bf/design-system/dist/future'
+import toast from 'react-hot-toast'
```

Comment:
when using shadcn, would just use the useToast hook instead of importing like that